package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "autosquash",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "license": "MIT",
   "files": [
+    ".github/workflows/autosquash.yml",
```

Comment:
Why publish this?

Reply:
Because it's [linked in the readme](https://github.com/tibdex/autosquash/blame/982dbef038e6eac178271383da91564e446efe4f/README.md#L19) and not packaging it would break the link in the [Marketplace page](https://github.com/marketplace/actions/autosquash).

Reply:
Yup

Reply:
It links to `master` for me.

Reply:
Oh yeah it's true but it would still break on pages such as this one https://github.com/tibdex/autosquash/tree/v2.0.1.

Reply:
I don't even think `npm pack` should include `README.md`, it is not essential to run the package. I could update the publish action to remove it manually.

Reply:
Use `v1.1.49` of `dylanvann/publish-github-action`. It won't include the readme, or other files npm includes by default (unless you add them to `files`).

See: https://github.com/DylanVann/publish-github-action/tree/releases/v1.1.49